packages/components/src/calendar/style.scss

```diff
@@ -24,19 +24,34 @@
 	}
 
 	.CalendarDay__selected_span {
-		background: $studio-woocommerce-purple;
+		background: theme(primary);
 		border: 1px solid $core-grey-light-700;
+
+		&:hover {
+			background: theme(secondary);
+			border: 1px solid $core-grey-light-500;
+		}
 	}
 
 	.CalendarDay__selected {
-		background: $studio-woocommerce-purple-70;
+		background: color(theme(primary) shade(20%));
```

Comment:
😮  This looks great in other themes too.

![Screen Shot 2020-06-17 at 2 21 59 PM](https://user-images.githubusercontent.com/1922453/84847792-fbdeda80-b0a5-11ea-92c5-7e69b38f14e0.png)